features/steps/payment_steps.py

```diff
@@ -0,0 +1,80 @@
+from behave import given, when, then
+from flask_api import status
+from app.db.interface import PaymentService
+import json
+
+@given(u'the following payments')
```

Comment:
Why all the unicode strings?  Are you doing so because it's part of the spec of `behave`?